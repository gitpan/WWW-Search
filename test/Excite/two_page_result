http://search.excite.com/search.gw?perPage=84&search=%2BJabba+%2Bbounty+%2Bhunter+%2BGreedo&start=0&showSummary=true&
http://search.excite.com/search.gw?perPage=84&search=%2BJabba+%2Bbounty+%2Bhunter+%2BGreedo&start=84&showSummary=true&
