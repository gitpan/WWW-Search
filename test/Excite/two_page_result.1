<HTML><HEAD>
<TITLE>Excite Search Results for "+Jabba +bounty +hunter +Greedo"</TITLE>
</HEAD>
<BODY bgcolor=ffffff>
<CENTER>
<FORM NAME=topbox ACTION=/search.gw>
<A HREF=http://www.excite.com/>
<IMG SRC=/img/exc/hdr/search.gif BORDER=0 WIDTH=600 HEIGHT=45 ALT="Excite Search" ISMAP USEMAP=#navstripe></A><BR>
<MAP NAME=navstripe>
<AREA SHAPE=rect COORDS=0,0,122,45 HREF=http://www.excite.com/>
<AREA SHAPE=rect COORDS=159,10,600,45 HREF=http://www.excite.com/>
</MAP>
<table border=0 cellpadding=1 cellspacing=0>
<tr><td bgcolor=cccccc>
<table border=0 cellspacing=0 cellpadding=4 width=598 bgcolor=ffffff><tr>
<td align=right bgcolor=ffffff>
<b><font face=arial size=-1>You searched for</font></b>
<input size=30 name=search value="+Jabba +bounty +hunter +Greedo">
<input type=hidden name=tsug value=-1>
<input type=hidden name=csug value=0>
</td>
<td><input type=submit value="Search Again"></td>
<td nowrap><font face=arial,helvetica size=-1><b>
<a href=/relocate/co=help;http://www.excite.com/Info/suggest.html>Help</a>
&nbsp;<br>
</b></font></td>
<td>&nbsp;</td>
</tr></table>
<table border=0 cellspacing=0 cellpadding=1 width=598>
<tr><td align=center bgcolor=ffcc00>
<font face=geneva,arial size=-1>
<font face=geneva,arial size=-1><b>
<font color=CC0000>Woof!</font> &nbsp;<a href="http://www.excite.com/relocate/co=searchresults_dogdecision;http://www7.excite.personalogic.com/common/qanda/pl.qanda;AZP6MRSe62UxHT5EZjsiA9hSPhVTWFTSfxVFFRLwckuAvtXEKFXsr1FskCg1$_qxVYerO17_nSmE$nKBifvOBJsxE$h0Wk90eo4HYd9UeT0dRDcS9lFfTHonx2lyUPVZqx_7N9rJUY9Qq6PVjsBiqNpv_qA0MUqV5J_rJheE0c8DjDzUDiosojgPWxM">Which dog is right for you?</A> | <a href="http://www.excite.com/relocate/co=searchresults_dogbuy;http://www.classifieds2000.com/cgi-cls/display.exe?primary_str=Dog&partner=excite&path=generic~pet~Search&">Find puppies for sale</A>
</b></font>
</font>
</td></tr>
</table>
</td></tr></table>
</form>
<small><br></small>
<a href="/relocate/ad=59202_cnet1007_stereo-wbsf.gif|pk=1431679;http://www.builder.com/live/?ad.excite.0.stereo" onMouseOver="window.status='Click Here !';return true" language="Javascript"><img src="http://image.eimg.com/ads/59202_cnet1007_stereo-wbsf.gif" ALT="Click Here !" HEIGHT=60 WIDTH=468 BORDER=1><CENTER><FONT SIZE=-1 FACE=geneva,arial COLOR=0000FF> <B> Click Here ! </B></FONT></CENTER></a>
</center>
<small><br></small>
<table border=0 cellpadding=0 cellspacing=0 width=100%><tr>
<td bgcolor=ffcc00 width=25%>
<table border=1 cellpadding=1 cellspacing=0 width=100%><tr>
<td align=center><font face=geneva,arial size=-1><b>
<a name=web>Web Results</a>
</b></font></td></tr></table>
</td><td valign=bottom width=100%>
<table border=0 cellspacing=0 cellpadding=0 width=100%>
<tr><td bgcolor=999999 width=100%><img src=/img/pixel/trans.gif height=2 width=1></td></tr></table>
</td></tr></table>
<table border=0 cellpadding=2 cellspacing=0 width=100%>
<tr><td width=40% nowrap><font face=arial>
Documents
&nbsp;85-105
</font></td>
<td align=right width=60% nowrap><font face=arial><b>
<a href="/search.gw?s=%2Bjabba+%2Bbounty+%2Bhunter+%2Bgreedo&c=web&start=84&showSummary=false">Show
Titles only</a>
&nbsp;
<a href="/search.gw?sort=site&c=web&s=%2Bjabba+%2Bbounty+%2Bhunter+%2Bgreedo">List
by Web site</a>
</b></font></td></tr></table>
<br>
<table border=0 cellpadding=2 cellspacing=1 width=120 align=right>
<tr>
<td align=center>
<table border="2" cellspacing="0" cellpadding="3" bordercolor="#336600" bgcolor="#fffcc" width=120>
<tr> <td align="center">
<b><font face="Verdana, Arial, Helvetica, sans-serif" size="-1">AMAZON.COM</font></b><br>
<font face="Verdana, Arial, Helvetica, sans-serif" size="-2">
<a href=/relocate/co=xsr_amazon;http://www.amazon.com/exec/obidos/excite-search/search=%2BJabba+%2Bbounty+%2Bhunter+%2BGreedo>Find Related Books</a><br>
Save up to 40%</font>
</td></tr></table>
</td>
</tr>
</table>
<SMALL>61% </SMALL>
<A HREF="http://www.worldaxes.com/anduril/dagdk.htm">UNOFFICIAL SW:CCG DAGOBAH DARK SIDE SPOILER LIST</A>&nbsp;
- UNOFFICIAL SW:CCG DAGOBAH DARK SIDE SPOILER LIST rev 5.0 11/06/97 TM &amp;® 1997 Lucasfilm Ltd. Star Wars and related marks are trademarks of Lucasfilm Ltd. Used under authorization by Decipher Inc. <BR><SMALL>http://www.worldaxes.com/anduril/dagdk.htm
<BR><A HREF="/search.gw?c=web&qbe=9988239&ctag=361AFF9D&showSummary=true">
Search for more documents like this one</A>
<P></SMALL>
<SMALL>58% </SMALL>
<A HREF="http://www.fanfix.com/stories/newrep/revenge.txt">Untitled</A>&nbsp;
- Title: Pqweeduk's Revenge Description: Pqweeduk the bounty hunter is out to revenge his brother Greedo's death. Determine to kill Han Solo or bring him and his family to Zorba the Hutt, Pqweeduk plans his attack. <BR><SMALL>http://www.fanfix.com/stories/newrep/revenge.txt
<BR><A HREF="/search.gw?c=web&qbe=38437425&ctag=361AFF9D&showSummary=true">
Search for more documents like this one</A>
<P></SMALL>
<SMALL>58% </SMALL>
<A HREF="http://www.gafm.com/starwars_nip.html">Goodnow's Action Figure Mart</A>&nbsp;
- Packaged Figures: Star Wars What's here? SALE ITEMS All new-in-package items are unopened unless otherwise noted. Maker / Year Price + shipping Kenner 1997 .00 green package w/hologram; non-mint <BR><SMALL>http://www.gafm.com/starwars_nip.html
<BR><A HREF="/search.gw?c=web&qbe=13690824&ctag=361AFF9D&showSummary=true">
Search for more documents like this one</A>
<P></SMALL>
<SMALL>58% </SMALL>
<A HREF="http://frodo.hiof.no/~deak/blooper/online/bloop31.txt">Untitled</A>&nbsp;
- These bloopers have been compiled from various documents, E-mail submissions and the readers of rec.arts.sf.starwars. The main documents used as source are _The Bad Guide to Star Wars_ (copyright Brandon Gillespie -- brandon@cc.usu.edu), the _Journal of the Bloopers_ (copyright Kevin Cox - Juan C. <BR><SMALL>http://frodo.hiof.no/~deak/blooper/online/bloop31.txt
<BR><A HREF="/search.gw?c=web&qbe=32484278&ctag=361AFF9D&showSummary=true">
Search for more documents like this one</A>
<P></SMALL>
<SMALL>58% </SMALL>
<A HREF="http://astro.caltech.edu/~rrg/starwars_coll.html">Star Wars Stuff</A>&nbsp;
- Star Wars Stuff Here's a list of recent Hasbro/Kenner Star Wars items in my collection: 3.5" Figures Card Type Coll.# Asst.# Trilogo Orange Luke Skywalker Long Saber Luke Skywalker Trilogo Orange <BR><SMALL>http://astro.caltech.edu/~rrg/starwars_coll.html
<BR><A HREF="/search.gw?c=web&qbe=36489337&ctag=361AFF9D&showSummary=true">
Search for more documents like this one</A>
<P></SMALL>
<SMALL>55% </SMALL>
<A HREF="http://www.bus.miami.edu/~jdavis/starwars/May-quotes.html">May Quotes</A>&nbsp;
- deak: he's too old to realize you'd just get in the way. -star wars rough draft found at http://www.wpi.edu/~ss/scripts_index.html along with many other drafts lord vinny &lt;cotewill@net-link.net &gt;Friday, May 31, 1996 at 21:51:43 (EDT) <BR><SMALL>http://www.bus.miami.edu/~jdavis/starwars/May-quotes.html
<BR><A HREF="/search.gw?c=web&qbe=29656494&ctag=361AFF9D&showSummary=true">
Search for more documents like this one</A>
<P></SMALL>
<SMALL>53% </SMALL>
<A HREF="http://empire.res.wabash.edu/nri/audio/charndex.htm">Star Wars Sounds - Character Arranged</A>&nbsp;
- Admiral Ackbar Bib Fortuna Boba Fett C-3PO Chewbacca Darth Vader Emperor Palpatine Grand Moff Tarkin Han Solo Jabba the Hutt Lando Calrissian Luke Skywalker Princess Leia Organa Obi-Wan Kenobi Other Characters and Sounds Other Droids Other Imperials Other Rebels R2-D2 Salacious Crumb Wedge Antilles Yoda <BR><SMALL>http://empire.res.wabash.edu/nri/audio/charndex.htm
<BR><A HREF="/search.gw?c=web&qbe=30972049&ctag=361AFF9D&showSummary=true">
Search for more documents like this one</A>
<P></SMALL>
<SMALL>53% </SMALL>
<A HREF="http://www.hamline.edu/~jstoller/class.html">Star Wars Classifieds</A>&nbsp;
- If you have a Star Wars home page I would apprecitate it if you made a link to my classifieds. I will make a link to yours on my link page. Thanks Due to the high volume of ads being summitted to the Ad page I can no longer guarantee your free ad will appear in a timley manner. <BR><SMALL>http://www.hamline.edu/~jstoller/class.html
<BR><A HREF="/search.gw?c=web&qbe=435390&ctag=361AFF9D&showSummary=true">
Search for more documents like this one</A>
<P></SMALL>
<SMALL>52% </SMALL>
<A HREF="http://www.fanfix.com/stories/newrep/dp2entir.txt">Untitled</A>&nbsp;
- ~ Date: Mon, 4 Sep 1995 15:21:12 -0500 (CDT) From: Alice Hadden To: Deborah657@aol.com And now, as if things couldn't get any darker...ali -------------------------------------------------------- Striving for the Light by Ali Hadden (hadden@mhd1.moorhead.msus.edu) Special thanks to Christine Bertz for her input and George Lucas for his, well, output! <BR><SMALL>http://www.fanfix.com/stories/newrep/dp2entir.txt
<BR><A HREF="/search.gw?c=web&qbe=41552615&ctag=361AFF9D&showSummary=true">
Search for more documents like this one</A>
<P></SMALL>
<SMALL>52% </SMALL>
<A HREF="http://www.primenet.com/~btn/faq/ptof.html">Action Figure Times -The Unofficial Star Wars POTF FAQ</A>&nbsp;
- The Exclusives Strikes Back! This variation FAQ is only as good as YOU make it. I do my observations, reporting and layout. I can't do it all without your help. I dedicate this FAQ to the memory of Steve Skelly, also known as former ASST. <BR><SMALL>http://www.primenet.com/~btn/faq/ptof.html
<BR><A HREF="/search.gw?c=web&qbe=1801740&ctag=361AFF9D&showSummary=true">
Search for more documents like this one</A>
<P></SMALL>
<CENTER> <TABLE BORDER=0 CELLPADDING=3 CELLSPACING=0 WIDTH=100%><TR>
<TD ALIGN=center BGCOLOR=FFFFCC>
<FONT FACE=arial><B>
<font face=geneva,arial size=-1><b>
<font color=CC0000>Woof!</font> &nbsp;<a href="http://www.excite.com/relocate/co=searchresults_dogdecision;http://www7.excite.personalogic.com/common/qanda/pl.qanda;AZP6MRSe62UxHT5EZjsiA9hSPhVTWFTSfxVFFRLwckuAvtXEKFXsr1FskCg1$_qxVYerO17_nSmE$nKBifvOBJsxE$h0Wk90eo4HYd9UeT0dRDcS9lFfTHonx2lyUPVZqx_7N9rJUY9Qq6PVjsBiqNpv_qA0MUqV5J_rJheE0c8DjDzUDiosojgPWxM">Which dog is right for you?</A> | <a href="http://www.excite.com/relocate/co=searchresults_dogbuy;http://www.classifieds2000.com/cgi-cls/display.exe?primary_str=Dog&partner=excite&path=generic~pet~Search&">Find puppies for sale</A>
</b></font>
</B></FONT>
</TD></TR></TABLE></CENTER><P>
<SMALL>52% </SMALL>
<A HREF="http://www.fanfix.com/stories/rebel/ord.txt">Untitled</A>&nbsp;
- ,mnnn This story is based on characters and situations created and owned by Lucasfilm, Ltd. No money is being made and no infringement is intended. Escape from Ord Mantell By Linda Bindner ``Cutting to...sublight engines.'' Han Solo reached his hand across the control panel and settled his fingers on the hydraulic levers that would bring the ship out of hyperspace. <BR><SMALL>http://www.fanfix.com/stories/rebel/ord.txt
<BR><A HREF="/search.gw?c=web&qbe=41605101&ctag=361AFF9D&showSummary=true">
Search for more documents like this one</A>
<P></SMALL>
<SMALL>51% </SMALL>
<A HREF="http://lonestar.texas.net/~cgregg/dark_dagobah_spoiler.html">UNOFFICIAL SW:CCG DAGOBAH DARK SIDE SPOILER LIST</A>&nbsp;
- Imperial Locations &middot;Anoat Dagobah U System Parsec: 5 If you control, all opponent's Ugnaughts on table are forfeit -1 If opponent initiates a battle here, your starships may move as a 'react' to or from nearest related asteroid sector. <BR><SMALL>http://lonestar.texas.net/~cgregg/dark_dagobah_spoiler.html
<BR><A HREF="/search.gw?c=web&qbe=19838175&ctag=361AFF9D&showSummary=true">
Search for more documents like this one</A>
<P></SMALL>
<SMALL>50% </SMALL>
<A HREF="http://www.gamecards.com/lists/starwars/Dagobah_Spoiler.html">Star Wars - Dagobah Spoiler List</A>&nbsp;
- HOME -GAME CARDS -TRADING CARDS -SHOP -CHECKLISTS -FAQ UNOFFICIAL SW:CCG DAGOBAH DARK SIDE SPOILER LIST TM &&(C) 1997 Lucasfilm Ltd. Star Wars and related marks are trademarks of Lucasfilm Ltd. Used under authorization by Decipher Inc. <BR><SMALL>http://www.gamecards.com/lists/starwars/Dagobah_Spoiler.html
<BR><A HREF="/search.gw?c=web&qbe=4776925&ctag=361AFF9D&showSummary=true">
Search for more documents like this one</A>
<P></SMALL>
<SMALL>48% </SMALL>
<A HREF="http://www.starwarsstore.com/encyclo.html">Encyclopedia of the Star Wars Universe</A>&nbsp;
- 367 North Magnolia Suite #103 El Cajon, Ca. 92020 Phone 619-588-7697 Fax 619-588-5682 E-Mail TheSWstore@aol.com 2-3KPR; A simple maintenance and diagnostics droid. Activates alarm sensors, security lighting, power fences on remote installations. <BR><SMALL>http://www.starwarsstore.com/encyclo.html
<BR><A HREF="/search.gw?c=web&qbe=10559663&ctag=361AFF9D&showSummary=true">
Search for more documents like this one</A>
<P></SMALL>
<SMALL>45% </SMALL>
<A HREF="http://members.tripod.com/~itsallgood_/allgood-2.html">It's All Good</A>&nbsp;
- The page is temporarily down, but if you click on the banner you will help contribute to getting the domain name we so deserve.. plus it's a great site! If you want another warezclan then help out and click on the banner at the top of the page! <BR><SMALL>http://members.tripod.com/~itsallgood_/allgood-2.html
<BR><A HREF="/search.gw?c=web&qbe=417815&ctag=361AFF9D&showSummary=true">
Search for more documents like this one</A>
<P></SMALL>
<SMALL>43% </SMALL>
<A HREF="http://www.nerf-herder.com/tuskentrader/lists/dausti.htm">The Tusken Trader Lists</A>&nbsp;
- Any Original Carded Figure not on this list. Luke X-Wing Fighter Gear .00 Luke Jedi Tan Vest (.00) either saber variants Boba Fett (.00 or .01) with half circles Certain loose freeze frames Admiral Ackbar w/hologram .00 C10 MOC <BR><SMALL>http://www.nerf-herder.com/tuskentrader/lists/dausti.htm
<BR><A HREF="/search.gw?c=web&qbe=19598139&ctag=361AFF9D&showSummary=true">
Search for more documents like this one</A>
<P></SMALL>
<SMALL>42% </SMALL>
<A HREF="http://splash.javasoft.com/beans/bdk_download.html">RACECALS - America's Favorite Racing Stickers</A>&nbsp;
- K12 loans financing students private <BR><SMALL>http://splash.javasoft.com/beans/bdk_download.html
<BR><A HREF="/search.gw?c=web&qbe=26210797&ctag=361AFF9D&showSummary=true">
Search for more documents like this one</A>
<P></SMALL>
<SMALL>41% </SMALL>
<A HREF="http://users.solve.net/~redding/sw/swccg/newslett/jedinew2.txt">Untitled</A>&nbsp;
- Returning from July's first issue is the head honcho, top banana, the one on the top, the supreme ruler of this newsletter, the great coming live, Steve55. Yes, it's him. Again joining us are writers TheLoneWof, Sabotaj, Ew41581, Roland768, Daischo, and Mjr Bludd. <BR><SMALL>http://users.solve.net/~redding/sw/swccg/newslett/jedinew2.t...
<BR><A HREF="/search.gw?c=web&qbe=25215182&ctag=361AFF9D&showSummary=true">
Search for more documents like this one</A>
<P></SMALL>
<SMALL>37% </SMALL>
<A HREF="http://www.duke.edu/~egm/prequel/spoil4.html">Star Wars Prequel FAQ</A>&nbsp;
- None Available
<BR><SMALL>http://www.duke.edu/~egm/prequel/spoil4.html
<BR><A HREF="/search.gw?c=web&qbe=8806829&ctag=361AFF9D&showSummary=true">
Search for more documents like this one</A>
<P></SMALL>
<SMALL>35% </SMALL>
<A HREF="http://www.primenet.com/~btn/ad.html">Action Figure Ads</A>&nbsp;
- Classified Ads Elsewhere-Action Figure Times: SEE BOTTOM OF PAGE FOR THE FIGURE GRADING SYSTEM. NOTICE: We now show the senders IP address. If you want to check out a person you can use it to track him down. <BR><SMALL>http://www.primenet.com/~btn/ad.html
<BR><A HREF="/search.gw?c=web&qbe=1158386&ctag=361AFF9D&showSummary=true">
Search for more documents like this one</A>
<P></SMALL>
<SMALL>33% </SMALL>
<A HREF="http://nerf-herder.com/potf2/figurescklist.html">Power of the Force 2 Individual Figure Checklist</A>&nbsp;
- Click the PRINT button on your browser Power of the Force 2 Figures .00 green card w/ or w/o hologram; Collection 2; Asst. #69605 .01 green card with hologram; stand-up type blister 4-LOM- .00 green card; Collection 2; Assortment #69605 8D8 Torture Droid Admiral Ackbar- <BR><SMALL>http://nerf-herder.com/potf2/figurescklist.html
<BR><A HREF="/search.gw?c=web&qbe=12982486&ctag=361AFF9D&showSummary=true">
Search for more documents like this one</A>
<P></SMALL>
<FORM ACTION="/search.gw" METHOD=GET>
<TABLE BORDER=0 CELLPADDING=0 CELLSPACING=0 WIDTH=100%>
<TR><TD>
<INPUT TYPE=hidden NAME=c VALUE=web>
<INPUT TYPE=hidden NAME=s VALUE="+Jabba +bounty +hunter +Greedo">
<INPUT TYPE=hidden NAME=showSummary VALUE=true>
<INPUT TYPE=hidden NAME=start VALUE=84>
<INPUT TYPE=hidden NAME=perPage VALUE=84>
<INPUT TYPE=submit NAME=prev VALUE="Previous Results">
</TD></TR></TABLE>
</FORM>
<small><a href="/relocate/co=webcraw;http://webcrawler.com/cgi-bin/WebQuery?text=%2BJabba+%2Bbounty+%2Bhunter+%2BGreedo&src=exciterecip">Try Your Search on WebCrawler</a></small><br>
<small><br></small>
<p>
<center>
<a href="/relocate/ad=59202_cnet1007_stereo-wbsf.gif|pk=1431679;http://www.builder.com/live/?ad.excite.0.stereo" onMouseOver="window.status='Click Here !';return true" language="Javascript"><img src="http://image.eimg.com/ads/59202_cnet1007_stereo-wbsf.gif" ALT="Click Here !" HEIGHT=60 WIDTH=468 BORDER=1><CENTER><FONT SIZE=-1 FACE=geneva,arial COLOR=0000FF> <B> Click Here ! </B></FONT></CENTER></a>
<small><br></small>
<table border=0 cellpadding=1 cellspacing=0>
<FORM NAME=botbox ACTION=/search.gw>
<tr><td bgcolor=cccccc>
<a name="botbox"> </a>
<table border=0 cellspacing=0 cellpadding=4 width=598 bgcolor=ffffff><tr>
<td align=right bgcolor=ffffff>
<b><font face=arial size=-1>You searched for</font></b>
<input size=30 name=search value="+Jabba +bounty +hunter +Greedo">
<input type=hidden name=tsug value=-1>
<input type=hidden name=csug value=0>
</td>
<td><input type=submit value="Search Again"></td>
<td nowrap><font face=arial,helvetica size=-1><b>
<a href=/relocate/co=help;http://www.excite.com/Info/suggest.html>Help</a>
&nbsp;<br>
</b></font></td>
<td>&nbsp;</td>
</tr></table>
</td></tr></table>
</form>
<small><br></small>
<p>
<font face=geneva,arial size=-1>
<nobr><b>More Search Services:</b>&nbsp;
<A HREF=http://www.excite.com/Info/searching.html>Search
Help</A> &nbsp; <A HREF=http://www.excite.com/search/options.html>Power Search</A>&nbsp;
<A HREF=http://nt.excite.com>Search the News</A></nobr><p>
<nobr><b>Services:</b>&nbsp;
<A HREF=http://www.excite.com/info/advertising/>Advertise on Excite</A>&nbsp; <A HREF=/relocate/co=xsr_botnav_bookflights;http://city.net/forms/reservations/>Book Flights</A>&nbsp;
<A HREF=/relocate/co=xsr_botnav_classifieds;http://classifieds2000.com/cgi-cls/display.exe?partner=excite&path=class&template=none&>Classifieds</A>&nbsp;
<A HREF=/relocate/co=7705;http://yellowpages.zip2.com>Yellow Pages</A>&nbsp;
<A HREF=http://www.excite.com/reference/>More...</A></nobr><p>
</font>
<font size=-2>
Copyright 1998 Excite Inc. &nbsp; Powered by <A HREF="http://www.sun.com">Sun Microsystems</A> &nbsp;
<A HREF=http://www.excite.com/info/disclaimer/>Disclaimer</A>
</font>
</center>
<IMG SRC="http://ad.preferences.com/ping.ng;spacedesc=TCExciteData_Excite_1x1_RunOfSite_Any&event=KeywordResults" HEIGHT=1 WIDTH=1 BORDER=0>
</body>
</html>
