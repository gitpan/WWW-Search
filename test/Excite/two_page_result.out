http://www.scifi.com/sfw/current/site.html
http://people.a2000.nl/tvdbrink/greedo.html
http://buteo.colorado.edu/~yosh/psi/system2/aliens/greedo/
http://www.theyknow.com/FB/jabba.html
http://www.imagesjournal.com/issue02/reviews/starwars.htm
http://www.geocities.com/Area51/Zone/6057/
http://www.nerf-herder.com/gswp/hide/theguys.html
http://www.angelfire.com/mn/Orlikowski/BountyHunters.html
http://www.personal.kent.edu/~nwillis/RPGBeedo.html
http://www-personal.umich.edu/~psholtes/StarWars/Starsounds.html
http://members.tripod.com/~stv1/index.html
http://geocities.com/Area51/Hollow/5915/starwars.html
http://www.world-net.net/users/kennyb/starwars.html
http://www.sirstevesguide.com/ac/ac.html
http://empire.res.wabash.edu/nri/char/han.htm
http://www.cae.wisc.edu/~ramsey/starpics.html
http://www.mygale.org/08/swritch/swtrivia.html
http://empire.res.wabash.edu/text/sw_notab.htm
http://www.decipher.com/starwars/productlibrary/cardlists/jabbaspalace/darkextra/beedox.htm
http://empire.res.wabash.edu/nri/char/wuher.htm
http://nerf-herder.com/swmhq/new.html
http://www.cinema1.com/movies97/returnofthejedi/us.html
http://www2.epix.net/~fnglk/sw_figs.html
http://www.fortunecity.com/roswell/jekyll/202/sex.html
http://people.mainz.netsurf.de/~engler/sw_anth.html
http://www.universal.nl/users/DWijnberg/encyd_l.htm
http://shell12.ba.best.com/~zzmaster/SF/star_wars_tales.html
http://www.ocslink.com/~au/Starwars.htm
http://www.personal.kent.edu/~nwillis/Quiz.html
http://www.nerf-herder.com/tuskentrader/lists/cwalke.htm
http://www.sayville.com/startrek.html
http://www.eisa.net.au/~echo3/characters/hunters2.html
http://www.swdatabase.com/pages/rotjbloopers.html
http://www.praetorian.net/io/lounge/cantina/people.html
http://www.princeton.edu/~nieder/sw/swthematic.html
http://www.roughcut.com/star_wars/main/stars.html
http://www.psextreme.com/excavate/mastertk.html
http://www.netwalk.com/~meena/starwars.html
http://nerf-herder.com/sbij/ts4.html
http://www.personal.kent.edu/~nwillis/CoolLinks.html
http://pilot.msu.edu/user/mccannr1/rules/swtse.txt
http://www.fanfix.com/stories/humor/wedgexpl.txt
http://www.j51.com/~tfth/swt.txt
http://www.inergy.com/troopers/
http://home.ptd.net/~dbohr/wars.html
http://www.amazon.com/exec/obidos/ASIN/0553564684/sitesthatwork
http://www.ea.pvt.k12.pa.us/htm/units/upper/clubs/scholium/2-14/phil2-14.htm
http://www.mindspring.com/~rosco29/rules/swtse.txt
http://www.geocities.com/Area51/Vault/6023/index.html
http://www.greatwest.ca/See/Issues/1997/970130/op.html
http://www.canoe.ca/JamStarWars/jan31_starwars_hobson.html
http://users.vnet.net/hehman/pinball/rules/sw.txt
http://cobweb.clemson.edu/~jvalice/starB.htm
http://www.columbia.edu/~mej16/sw.html
http://www.bama.ua.edu/~mcphe002/sw.htm
http://www3.sk.sympatico.ca/earlr/page4.html
http://members.tripod.com/~balki33/weapons.html
http://www.primenet.com/~btn/afun/afunny.html
http://www.yakface.com/ysnn/newsarchives.htm
http://www.primenet.com/~btn/area.html
http://www.jedinet.com/literature/raves.htm
http://www.cis.ohio-state.edu/text/faq/usenet/starwars/www/part3/faq.html
http://lonestar.texas.net/~cgregg/anh-d.txt
http://www.innercite.net/~rmouse/humor/year97/9.6.97.html
http://pw2.netcom.com/~kestral/Wes2.html
http://www.personal.kent.edu/~psekula/figures.html
http://www.acsu.buffalo.edu/~rwleach/swtoys.html
http://users.solve.net/~redding/sw/swccg/newslett/deckhlp4.txt
http://www.mufty.com/swccg/interviews/robsilmser.html
http://www.yakface.com/ysnn/newsarchives1.htm
http://www.corona.bc.ca/films/details/sw1.html
http://www.stwing.upenn.edu/starwars/films/quizbook.txt
http://www.mufty.com/swccg/interviews/edmiller.html
http://www.wiseacres.com/feb.htm
http://www.starwars.hasbro.com/low/toy/listp.html
http://www.worldaxes.com/anduril/jabbadk.htm
http://www.shavenwookie.com/swlinksmedia.html
http://hot.virtual-pc.com/mufty/swccg/interviews/tonycase.html
http://www.csi.ull.es/alumnado/alu339/journal5.txt
http://www.thevision.net/dread/starwars/sounds.htm
http://www.egosystem.com/starwars/rotj.html
http://freespace.virgin.net/chris.harper1/starwars/complete.htm
http://www.wiseacres.com/sep.htm
http://lonestar.texas.net/~cgregg/dark_anh_spoiler.html
http://www.worldaxes.com/anduril/dagdk.htm
http://www.fanfix.com/stories/newrep/revenge.txt
http://www.gafm.com/starwars_nip.html
http://frodo.hiof.no/~deak/blooper/online/bloop31.txt
http://astro.caltech.edu/~rrg/starwars_coll.html
http://www.bus.miami.edu/~jdavis/starwars/May-quotes.html
http://empire.res.wabash.edu/nri/audio/charndex.htm
http://www.hamline.edu/~jstoller/class.html
http://www.fanfix.com/stories/newrep/dp2entir.txt
http://www.primenet.com/~btn/faq/ptof.html
http://www.fanfix.com/stories/rebel/ord.txt
http://lonestar.texas.net/~cgregg/dark_dagobah_spoiler.html
http://www.gamecards.com/lists/starwars/Dagobah_Spoiler.html
http://www.starwarsstore.com/encyclo.html
http://members.tripod.com/~itsallgood_/allgood-2.html
http://www.nerf-herder.com/tuskentrader/lists/dausti.htm
http://splash.javasoft.com/beans/bdk_download.html
http://users.solve.net/~redding/sw/swccg/newslett/jedinew2.txt
http://www.duke.edu/~egm/prequel/spoil4.html
http://www.primenet.com/~btn/ad.html
http://nerf-herder.com/potf2/figurescklist.html
